Basic-Car-Maintenance/Documentation.docc/getting-started.tutorial

```diff
@@ -0,0 +1,103 @@
+@Tutorial(time: 15) {
+    @Intro(title: "How to contribute to this project.") {
+        Here you will find instructions on how to prepare, fork, clone, configure, build and contribute to this project. 
+        
+    }
+    
+    @Section(title: "Prerequisites") {
+
+        @ContentAndMedia {
+            Add engaging section text here.
+        }
+        
+        @Steps {
+            @Step {
+                Read the Code of Conduct
+            }
+            @Step {
+                Read the CONTRIBUTING guidelines
+            }
+            @Step {
+                Download Xcode 15 or later
+            }
+            @Step {
+                Install Github command line tools
+            }
+
+        }
+    }
+            
+    @Section(title: "Fork and Clone") {
+        @ContentAndMedia {
+            Fork and clone 
+            
+            @Image(source: 2-fork.png, alt: "GitHub Fork")
+        }
+        
+        @Steps {
+            @Step {
+                Fork the repo on GitHub to your own account.  Select Fork from the Basic-Car-Maintenance github page.
+                
+                @Image(source: 2-fork.png, alt: "GitHub Fork")
+            }
+            
+            @Step {
+                Clone your fork to your computer. 
+                
+                gh repo clone USERNAME/Basic-Car-Maintenance
+                
+                @Image(source: 3-clone.png, alt: "CLI github clone")
+            }
+        }
+    }
+    @Section(title: "Configure and Build") {
+        @ContentAndMedia {
```

Comment:
Can you edit the description to: "Configuring the project with the xcconfig files to build the project on your machine"